.github/workflows/checks.yml

```diff
@@ -208,12 +208,14 @@ jobs:
     strategy:
       fail-fast: false
     env:
-      INTEGRATION_RPC_URL: ${{ secrets.INTEGRATION_RPC_URL }}
-      TESTNET_RPC_URL: ${{ secrets.TESTNET_RPC_URL }}
-      INTEGRATION_ACCOUNT_PRIVATE_KEY: ${{ secrets.INTEGRATION_ACCOUNT_PRIVATE_KEY }}
-      INTEGRATION_ACCOUNT_ADDRESS: ${{ secrets.INTEGRATION_ACCOUNT_ADDRESS }}
-      TESTNET_ACCOUNT_PRIVATE_KEY: ${{ secrets.TESTNET_ACCOUNT_PRIVATE_KEY }}
-      TESTNET_ACCOUNT_ADDRESS: ${{ secrets.TESTNET_ACCOUNT_ADDRESS }}
+      GOERLI_INTEGRATION_RPC_URL: ${{ secrets.INTEGRATION_RPC_URL }}
+      GOERLI_TESTNET_RPC_URL: ${{ secrets.TESTNET_RPC_URL }}
+      SEPOLIA_INTEGRATION_RPC_URL: ${{ secrets.SEPOLIA_INTEGRATION_RPC_URL }}
```

Comment:
just to be sure - these secrets were already added to repo, right?

Reply:
Yes, these secrets have already been set in the repo. If they weren't the `Tests on networks` would fail.